MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/AddNewEntryManually.java

```diff
@@ -0,0 +1,149 @@
+package com.izor066.android.mediatracker.ui;
+
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.inputmethod.EditorInfo;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.izor066.android.mediatracker.MediaTrackerApplication;
+import com.izor066.android.mediatracker.R;
+import com.izor066.android.mediatracker.api.model.Book;
+import com.izor066.android.mediatracker.ui.fragment.DatePickerFragment;
+import com.squareup.picasso.Picasso;
+
+public class AddNewEntryManually extends AppCompatActivity implements DatePickerFragment.OnDatePubSetListener, Button.OnClickListener, TextView.OnEditorActionListener {
+
+    String TAG = getClass().getSimpleName();
+
+    private final String IMAGE_PLACEHOLDER = "https://s.gr-assets.com/assets/nophoto/book/blank-133x176-8b769f39ba6687a82d2eef30bdf46977.jpg";
+
+    String mAddTitle ="";
```

Comment:
All of the fields can be `private` as they're not, and shouldn't, be accessed outside of this class.